packages/suite/src/views/wallet/transactions/components/TokenList/index.tsx

```diff
@@ -97,18 +114,15 @@ const TokenList = ({ tokens, explorerUrl, isTestnet }: Props) => {
                     return (
                         <Fragment key={t.address}>
                             <Col isTestnet={isTestnet}>
-                                <TokenName>
+                                <TokenNameWrapper>
                                     <TokenSymbol>{t.symbol?.toUpperCase()}</TokenSymbol>
-                                    {` - ${t.name}`}
-                                </TokenName>
+                                    <TokenName> - {t.name}</TokenName>
```

Comment:
space before `-` will not work anymore is it intended?

Reply:
yup, I think it didn't even worked before, there is margin-right on `TokenSymbol`